module-03-react/src/components/Gallery.jsx

```diff
@@ -0,0 +1,146 @@
+import React, {
+    useEffect,
+    useState,
+    useImperativeHandle,
+    forwardRef,
+} from "react";
+import axios from "axios";
+import "/src/styles/components.css";
+import Card from "./Card.jsx";
+
+const baseURL = "https://swapi.dev/api/";
+
+const Gallery = forwardRef(
```

Comment:
The use of `forwardRef` will soon be obsolete with React 19's upcoming changes, which will allow function components to receive `ref` directly as a prop. Consider refactoring this component to prepare for this change